### ISSUE TITLE
!players

### ISSUE THREAD

**Issue by jun1024** [egebilecen/PZServerDiscordBot#57]
I have a suggestion about new feature, can you make a command for checking current user number? I used to type 'players' in startserver64.bat cmd, but I can't type it in cmd anymore with your bot. so It would be great if you add a checking player number command which I can use in discord channel.

**Comment by egebilecen** [comment]
Yeah, I will look into that. No exact time when it will be added though.

**Comment by egebilecen** [comment]
Hey, for a quick temporary fix, I added `!server_cmd` command which you can directly pass commands to the server. You can now simply write `!server_cmd players` then check the server console. I will add this feature to the bot as `!player_count` or something but I don't know when so this temporary fix should be good for now.

**Comment by jun1024** [comment]
That was all I need, thanks.

**Comment by reinierssp** [comment]
> Oye, para una solución temporal rápida, agregué un comando que puede pasar comandos directamente al servidor. Ahora puede simplemente escribir y luego verificar la consola del servidor. Agregaré esta función al bot como o algo así, pero no sé cuándo, así que esta solución temporal debería ser buena por ahora.`!server_cmd``!server_cmd players``!player_count`

The command only works in the console but I would like it to be represented in Discord, especially in the general channel and if possible show not the quantity but the names of the users.
